src/main/java/mate/dao/jdbc/ManufacturerJdbcImpl.java

```diff
@@ -0,0 +1,118 @@
+package mate.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.dao.ManufacturerDao;
+import mate.exception.DataProcessingException;
+import mate.lib.Dao;
+import mate.model.Manufacturer;
+import mate.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerJdbcImpl implements ManufacturerDao {
+
+    private static Optional<Manufacturer> createNewManufacturer(ResultSet resultSet) {
+        try {
+            Manufacturer manufacturer = new Manufacturer(resultSet.getString("manufacturer_name"),
+                    resultSet.getString("manufacturer_country"));
+            manufacturer.setId(resultSet.getLong(1));
```

Comment:
resultSet.getLong -?
did you read a checklist? - https://mate-academy.github.io/jv-program-fulltime/02_jdbc/checklist/05_JBDC_checklist.html